introduction.md

```diff
@@ -0,0 +1 @@
+Hi, my name is Carlos Ardila. I'm 35 years old. I love software development, actually I am developing in Java, .Net and in Unity.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.